src/components/createConnect.js

```diff
@@ -199,6 +201,10 @@ export default function createConnect(React) {
         }
 
         render() {
+          if (isStateless) {
+            return <WrappedComponent {...this.nextState} />;
+          }
+
```

Comment:
This should probably not be here. If we decide to go ahead with this, we should probably create render above and apply it to this class.